roles/os_net_setup/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ cifmw_os_net_setup_config:
     subnets:
       - name: public_subnet
         cidr: 192.168.122.0/24
-        allocation_pool_start: 192.168.122.201
+        allocation_pool_start: 192.168.122.190
```

Comment:
May be good to increase it bit further, 190 -> 171?

And also adjust https://github.com/openstack-k8s-operators/ci-framework/blob/7caff677998e13118cff72a9ba4bb956b5f072c0/scenarios/reproducers/networking-definition.yml#L20 to 170 to match length https://github.com/openstack-k8s-operators/ci-framework/blob/7caff677998e13118cff72a9ba4bb956b5f072c0/scenarios/reproducers/networking-definition.yml#L121